jll/C/CURL_jll/Compat.toml

```diff
@@ -11,7 +11,7 @@ Artifacts = "1"
 Libdl = "1"
 
 ["8-8.5"]
-LibCURL_jll = "8.5.0-8"
+LibCURL_jll = "8.5.0-8.5"
```

Comment:
Can probably also allow 8.6 here
```suggestion
LibCURL_jll = "8.5.0-8.6"
```

Reply:
I thought they have to match precisely?

Reply:
I thought so too, but if no one noticed big problems before maybe we have some flexibility 😛 we can further tighten the compat later if necessary, but at least we avoid the completely incompatible combination with this PR.